models/cooperation.js

```diff
@@ -0,0 +1,38 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { COOPERATION_STATUS }
+} = require('~/consts/validation')
+
+const cooperationSchema = new Schema(
+  {
+    offerId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Offer',
+      required: true
+    },
+    tutorId: {
```

Comment:
gotta think about those fields tho
prolly it makes sense to name them differently so its clear who initiated that cooperation